ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml

```diff
@@ -186,7 +186,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FORCE_SUCCESS_EXIT: "no"
       TEST_FILTERS: ~ChkUpgrade&;~ConnectedOnly&;Smokerun&
-      TEST_SCENARIOS: Windows_Containers
```

Comment:
Please, remember to restore back the Windows_Containers scenario ;-)

Reply:
yes. after debugging is done, I will recover it.